plugins/chat/assets/javascripts/discourse/services/chat-threads-manager.js

```diff
@@ -0,0 +1,70 @@
+import Service, { inject as service } from "@ember/service";
```

Comment:
Glad that you seem to like the pattern and use it 👌

Reply:
Yeah it's very handy, I like the `@service blah` syntax, also I have no idea of any other way of doing this so that's why I copy it haha.